network/p2p/p2p_pubsub.go

```diff
@@ -238,6 +241,15 @@ func (n *p2pNetwork) handlePubsubMessages(logger *zap.Logger) func(ctx context.C
 			return errors.New("message was not decoded")
 		}
 
+		signedSSVMessage := &spectypes.SignedSSVMessage{}
+		if err := signedSSVMessage.Decode(msg.GetData()); err != nil {
+			return fmt.Errorf("signed message was not decoded: %w", err)
+		}
+		decodedMsg, err := queue.DecodeSignedSSVMessage(signedSSVMessage)
+		if err != nil {
+			return fmt.Errorf("message was not decoded: %w", err)
+		}
+
```

Comment:
why do we need this if we didnt need this before? (for example on `main` branch)

Reply:
yeah, @nkryuchkov clarified here, removed.